src/DotLiquid/StandardFilters.cs

```diff
@@ -931,6 +931,51 @@ public static IEnumerable Compact(object input)
             ary.RemoveAll(item => item == null);
             return ary;
         }
+
+        /// <summary>
+        /// Creates an array including only the objects with a given property value, or any truthy value by default.
+        /// </summary>
+        /// <param name="input">an array to be filtered</param>
+        /// <param name="propertyName">The name of the property to filter by</param>
+        /// <param name="target_value">Value to retain, if null object containing this property are retained</param>

```

Comment:
"Value to retain, if null object containing this property are retained"

Null objects would not be retained, would they?

This parameter description is likely wrong. The target value specifies what value the matched named property must have for the containing objects to be selected by the filter.

Reply:
Comment has been updated